Manuscript/Text/02F_mutein.md

```diff
@@ -1,3 +1,9 @@
 ## Model Accurately Captures Cell Type-Specific Response to IL-2 Muteins
 
 ![**Model and tensor factorization predicts and decodes cell-type specific responses to IL-2 muteins.** A) IL-2Rα and IL-2Rβ/γ~c~ dissociation constants for panel including wild-type IL-2 and IL-2 muteins. B-C) Predicted and measured T~reg~ activation specificity compared to NK (B) and T helper (C) cells. D) Partial derivatives of T~reg~ activation specificity compared to NK and T helper cells with respect to each surface and endosomal reverse binding rate constant. E) T~reg~ activation specificity with respect to NK and T helper cells as a function of IL-2Rβ/γ~c~ binding affinity for ligands with wild type and reduced IL-2Rα affinity. Specificity values are shown for cells exposed to a cytokine concentration of 38 pM. F) Experimentally measured and model-predicted EC~50~s of dose responses across IL-2 muteins and all 10 cell types. EC~50~s are shown for 1 hr time point. G-J) Tensor factorization of experimentally measured cellular activation values for wild-type IL-2/15, and IL-2 muteins. Component values versus ligand (G), cell type (H), time (I), and cytokine concentration (J). ](./Figures/figure6.svg){#fig:mutFac}
+
+In order to further evaluate our model, we experimentally measured PBMC response to several IL-2 muteins and Fc-bound IL-2 monomers. Muteins were created by introducing targeted point mutations to IL-2 domains known to be instrumental in determining the binding affinity of the cytokine to both IL-2Rα and IL-2Rβ/γ~c~ (insert paper(s) here). Another way in which cytokine binding affinities can be modified is by conjugation of another domain to the cytokine. In clinical applications, cytokines are often conjugated to antibodies in order to increase the longevity of cytokine-based drugs in the body. Conjugating these antibodies to the N or C-terminus of the cytokine has differing effects on the cytokine's binding affinities. We quantified the effect of these mutations and conjugations on IL-2Rα and IL-2Rβ/γ~c~ receptor binding affinities, and found that conjugation of Fc domains to the N-terminus selectively lowered IL-2Rβ/γ~c~ affinity, while conjugation to the C-terminus selectively lowered IL-2Rα affinity ([@fig:mutFac]A). Using these altered affinities, we were able to accurately predict cell-type specific activation levels when PBMC cells were exposed to these modified ligands ([@fig:supp6]).
```

Comment:
I wouldn't say Fc conjugation was known to affect affinity. I think you can just remove that sentence and start with "In clinical applications..."

Reply:
Before transitioning, something along the lines of: "Therefore, Fc conjugation can either cooperate or hinder mutation-mediated changes in receptor affinity, and affinity should be assessed in the clinical format."